src/component/common/ConstraintAccordion/ConstraintAccordionView/ConstraintAccordionViewHeader/ContraintAccordionViewHeaderMultipleValues/ConstraintAccordionViewHeaderMultipleValues.tsx

```diff
@@ -50,6 +51,21 @@ export const ConstraintAccordionViewHeaderMultipleValues = ({
     return (
         <div className={styles.headerValuesContainerWrapper}>
             <div className={styles.headerValuesContainer}>
+                <ConditionallyRender
+                    condition={
+                        'result' in constraint && !Boolean(constraint.result)
```

Comment:
Could put these conditionals in a helper function, or something like that, for readability.